zulipterminal/ui_tools/views.py

```diff
@@ -899,6 +899,14 @@ def __init__(self, controller: Any, title: str, *, zt_version: str,
         super().__init__(controller, widgets, 'ABOUT', popup_width, title)
 
 
+class MsgLinksView(PopUpView):
```

Comment:
The related issue, #618, and the discussion [here](https://chat.zulip.org/#narrow/stream/206-zulip-terminal/topic/links) mentioned that we would want links to be listed at the end (with a different style) and combined with the `MsgInfoView`. 

On the contrary, this doesn't add links in `MsgInfoView` considering how 'Reactions' can potentially take a significant amount of space in the view and how it would be easier to work with a popup [than trailing links in `MessageBox`] later (e.g. handling internal/external links). Thoughts?

Reply:
I'd suggest for a v1 we can integrate links as the end of a message, and also as a category above reactions in the message-info-view. I don't expect too many links in a single message?